src/c18n/README.md

```diff
@@ -1,5 +1,13 @@
 # Library compartmentalization
 
+<div class="warning">
+This material is obsolete.  Among other things,
+<code>/libexec/ld-elf-c18n.so.1</code> no longer exists.
+See the latest
+<a href="https://man.cheribsd.org/cgi-bin/man.cgi/dev/c18n">c18n(3) man page</a>
```

Comment:
Is dev really what we want to link?

Reply:
Not sure how I ended up with that link. The default search lands on the latest release which is probably what we want.